### PR TITLE
display that a comment was edited and provide hover-over tooltip

### PR DESCRIPTION
**Pull request by johnquiwa**
### Assignee Tasks

https://github.com/yungtiec/bkp-alpha/issues/75
- As a user, after posting, (but before others have replied), I want to be able to edit my comment (and it can say "edited" like in telegram, or slack)

### CODE REVIEW

File: client/scenes/document/scenes/Version/components/CommentContainer/MainComment.js [hunk 1]
```diff
@@ -35,6 +34,7 @@ export default ({
     (comment, url) => comment.replace(new RegExp(url, "g"), `[${url}](${url})`),
     comment.comment
   );
+  const wasEdited = comment.updatedAt === comment.createdAt;
```
Comment (yungtiec) [inline client/scenes/document/scenes/Version/components/CommentContainer/MainComment.js]: !==
'edited' is showing in unedited comments

Comment (johnquiwa) [reply client/scenes/document/scenes/Version/components/CommentContainer/MainComment.js]: Whoops, thanks for catching that. Updated it!

Comment (johnquiwa) [reply client/scenes/document/scenes/Version/components/CommentContainer/MainComment.js]: And ohh interesting. Do you think I should add logic to check the reviewed column? Also, are users able to edit after a comment has been verified?

Comment (yungtiec) [reply client/scenes/document/scenes/Version/components/CommentContainer/MainComment.js]: I'll say not to worry this for now. We might even remove the verify button. We mostly just mark a comment as spam, and the comment will not be displayed anymore.

**Pull request merged**